tsdb/engine/wal/wal.go

```diff
@@ -106,9 +109,6 @@ var (
 	// ErrCompactionRunning to return if we attempt to run a compaction on a partition that is currently running one
 	ErrCompactionRunning = errors.New("compaction running")
 
-	// ErrCompactionBlock gets returned if we're reading a compressed block and its a file compaction description
```

Comment:
It's an internal marker error. I changed it to return the named block's filename instead of an error.